packages/block-library/src/post-template/style.scss

```diff
@@ -30,3 +30,9 @@
 		}
 	}
 }
+
+@media ( max-width: $break-small ) {
+	.wp-block-post-template-is-layout-grid.wp-block-post-template-is-layout-grid.wp-block-post-template-is-layout-grid.wp-block-post-template-is-layout-grid {
```

Comment:
Going with a block-specific media query for now to copy the current responsive behaviour. This selector is _far from_ ideal though. I wonder if, now that layout rule specificity has been reduced, it would also be possible to reduce specificity for the `wp-container-x` selector? Ideally though, we'd update the style engine to process media queries so they can be added as part of the layout support (also in the future we should look into moving to container queries).

Reply:
> Ideally though, we'd update the style engine to process media queries so they can be added as part of the layout support

This is an interesting thought. I think it shouldn't be too hard to support media/container queries in the future.

I've added an item to the [style engine project board](https://github.com/orgs/WordPress/projects/19).